_schemaV2.graphql

```diff
@@ -2224,46 +2226,44 @@ type ArtistSeries {
     priceRange: String
     saleID: ID
     size: Int
+
+    # Filter results by Artwork sizes
     sizes: [ArtworkSizes]
     sort: String
     tagID: String
     width: String
   ): FilterArtworksConnection
   forSaleArtworksCount: Int!
+
+  # A globally unique ID.
+  id: ID!
   image: Image
-  imageHeight: Int
```

Comment:
These were only used internally to resolve the `image` field which now has proper support sans stitching.